bentoml/_internal/bento/docker/entrypoint.sh

```diff
@@ -11,7 +11,7 @@ _is_sourced() {
 
 _main() {
 	# if first arg looks like a flag
-	if [ "${1:0:1}" = '-' ]; then
+	if [ -z "$@" ] || [ "${1:0:1}" = '-' ]; then
```

Comment:
Is `-z "$@"` a superset of `"${1:0:1}" = '-'`?

Reply:
Not really

Reply:
`command`: 
`-z "$@"` -> true 
`[ "${1:0:1}" = '-' ]` -> false

Reply:
Minor style nit, but I think `[ $# -eq 1 ]` is clearer than `[ -z "$@" ]`.